cogboard-webapp/src/components/BoardForm/index.js

```diff
@@ -11,6 +11,18 @@ const BoardForm = ({ renderActions, ...initialFormValues }) => {
   const { values, handleChange } = useFormData(initialFormValues);
   const { autoSwitch } = values;
 
+  const handleInput = (e) => {
+    const inputValue = e.target.value;
+    let intValue = isNaN(parseInt(inputValue)) ? COLUMNS_MIN : parseInt(inputValue);
+    intValue = intValue > COLUMNS_MAX ? COLUMNS_MAX : intValue;
+    intValue = intValue < COLUMNS_MIN ? COLUMNS_MIN : intValue;
+    e.target.value = intValue.toString();
+  };
+
+  const columnsPlaceholder = () => {
+    return COLUMNS_DEFAULT.toString();
```

Comment:
String() it is better for this operation, because you want to cast primitive value to a string. Using toString() js compiler first convert primitive number to it's object representation and then back to string. The same is with Number().